series/supportedseries_linux_test.go

```diff
@@ -102,9 +102,9 @@ func (s *supportedSeriesSuite) TestSupportedJujuControllerSeries(c *gc.C) {
 	filename := filepath.Join(d, "ubuntu.csv")
 	err := ioutil.WriteFile(filename, []byte(distInfoData), 0644)
 	c.Assert(err, jc.ErrorIsNil)
-	s.PatchValue(series.DistroInfo, filename)
+	s.PatchValue(series.UbuntuDistroInfoPath, filename)
 
-	expectedSeries := []string{"bionic", "disco", "eoan", "xenial"}
```

Comment:
We should patch the `time.Now()` in `updateLocalSeriesVersions` as the package goes out of date with every passing day. The tests expect dates to be at a certain time.